quant.info.yml

```diff
@@ -7,4 +7,5 @@ core_version_requirement: ^8 || ^9
 core: 8.x
 
 dependencies:
+  - drupal:token
```

Comment:
I think technically this dependency should go into `quant_search` -- the main quant module works fine without token

Reply:
Good point, I'll move.